### PR TITLE
Separate edw definitions from icu definitions

### LINKED ISSUE #1109: Move EDW definitions to standalone script in /definitions

### PR DESCRIPTION
**Pull request by raimonpv**
## Issues closed
Closes #1109 

## Changelog
- [x] Moves EDW definitions to standalone script in /definitions

## Checklist
- [x] The following are complete:
    - Added or revised tests if appropriate.
    - If new packages are installed, `requirements.txt` is updated.
    - If documentation should be updated, a new issue was made in `ml4c3-wiki`.
    - The PR name describes its content and does not contain "Ref #".
    - The PR does not add PHI.

### CODE REVIEW

File: definitions/icu.py [hunk 1]
```diff
@@ -2,8 +2,8 @@
 import os
 from typing import Any, Dict, List
 
-EDW_PREFIX = "edw"
-EDW_EXT = ".csv"
+VISUALIZER_PATH = os.path.join(os.path.abspath(__file__ + "/../.."), "visualizer")
```
Comment (erikr) [inline definitions/icu.py:5]: Should we move this to https://github.com/aguirre-lab/ml4c3/blob/b546ab6e6bcd08107ef6dd13a213613749a98ee9/visualizer/properties.py#L8 and not have to import anything in `properties.py`?

File: ingest/icu/readers.py [hunk 2]
```diff
@@ -14,13 +14,8 @@
...
 # Imports: first party
 from ml4c3.utils import get_unix_timestamps
-from definitions.icu import (
-    EDW_FILES,
-    MED_ACTIONS,
-    ALARMS_FILES,
-    ICU_SCALE_UNITS,
-    MAPPING_DEPARTMENTS,
-)
+from definitions.edw import EDW_FILES, MED_ACTIONS
```
Comment (erikr) [inline ingest/icu/readers.py:17]: This is temporary, yes? Eventually we want:
```
ingest/bedmaster/
```

that should not import anything EDW-related.

I don't think we have any EDW ingest code.

**Review by erikr — COMMENTED** [review]
Raimon, great work cleaning this up.

I think we should rename `ingest/icu` to `ingest/bedmaster` to be consistent with naming subdirectories after data modalities.

Also, ideally no code in `ingest/bedmaster` imports anything EDW-related, e.g. `from definitions.edw import SOMETHING`.

What do you think?

**Comment by raimonpv** [comment]
I think that we should rename it once it is completely separated from EDW, right now it still contains the tensorization of both EDW and Bedmaster. About Bedmaster importing definitions from EDW I don't have a clear idea of what we should do, as Bedmaster relies on EDW information to cross-reference the data.

**Review by erikr — APPROVED** [review]

**Pull request merged**